qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -823,23 +823,23 @@ test_status_in_pool_dir()
     local description_pattern="$5"
     local service_pattern="$6"
 
-    local status_log=${TEMPDIR}/$(mkfname ${cluster}-${pool}-${image}.mirror_status)
-    CEPH_ARGS='' rbd --cluster ${cluster} mirror image status ${pool}/${image} |
-        tee ${status_log} >&2
-    grep "^  state: .*${state_pattern}" ${status_log} || return 1
-    grep "^  description: .*${description_pattern}" ${status_log} || return 1
+    local status=$(CEPH_ARGS='' rbd --cluster ${cluster} mirror image status \
+                     ${pool}/${image})
+    echo "$status" | grep "^  state: .*${state_pattern}"  || return 1
```

Comment:
A pipe might be considered excessive for this:
```suggestion
    grep "^  state: .*${state_pattern}" <<< "$status" || return 1
```